third-party/components/ApprovalCommentForm/ApprovalCommentForm.js

```diff
@@ -262,17 +206,18 @@ class ApprovalCommentForm extends React.Component<Props, State> {
                         {/* TODO: Replace Draft.js in box-react-ui implementation of ApprovalCommentForm */}
                         <TextArea
                             className='bcs-comment-input'
-                            isRequired={isOpen}
                             isDisabled={isDisabled}
                             name='commentText'
-                            onChange={this.onMentionSelectorChangeHandler}
+                            label='Annotation Comment'
```

Comment:
do we need to worry about localized labels?

Reply:
The TextArea component in BRUI requires a label